### ISSUE TITLE
Add arrow marker plot

### ISSUE THREAD

**Issue by anitagraser** [movingpandas/movingpandas#332]
Add plot type to show individual trajectory locations / fixes and the corresponding movement direction with arrow markers

e.g. arrow markers based on cog column: 

```
tmp['dash_angle'] = ((tmp.cog*-1.0)+90).astype(float)
tmp['triangle_angle'] = ((tmp.cog*-1.0)).astype(float)

tmp.hvplot(geo=True, c='sog', marker='dash', angle=dim('dash_angle'), size=200, line_width=2, hover_cols=['cog','sog','dash_angle']) * 
tmp.hvplot(geo=True, c='sog', marker='triangle', angle=dim('triangle_angle'), size=100, hover_cols=['cog','sog','triangle_angle']) *
```

**Note:** The hover_cols are important, otherwise, you'll get the error: "Specified angle dim transform dim('triangle_angle') could not be applied, as not all dimensions could be resolved."

![image](https://github.com/movingpandas/movingpandas/assets/590385/10fad483-50f3-49bc-9605-e806876c4ccf)

**Comment by anitagraser** [comment]
See: https://github.com/movingpandas/movingpandas/pull/340

**Comment by anitagraser** [comment]
![image](https://github.com/movingpandas/movingpandas/assets/590385/5c422073-4d0b-45c3-bc6b-185a46d13b61)

**Comment by anitagraser** [comment]
Implemented in [#fbab0e0](https://github.com/movingpandas/movingpandas/commit/fbab0e01a1bb5f110683b54486e52de42036a38e)
